api/database/seeders/ClassificationRolesSeeder.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+
+class ClassificationRolesSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+      $roles = [
+            [
+                'key' => 'technician ',
```

Comment:
Since the ClassificationRole is so closely tied with role, I reccomend including the role in the `key`.
```suggestion
                'key' => 'technician_it01',
```